app/pods/publications/publication/documents/template.hbs

```diff
@@ -198,40 +198,39 @@
 </div>
 
 {{#if this.isOpenPieceUploadModal}}
-  <WebComponents::VlModal
-    {{!-- @closeModal={{perform this.cancelUploadPieces}} --}}
-    @closeModal={{this.cancelUploadPieces}}
-    @isOverlay={{true}}
-    @large={{true}}
-    @title={{t "documents-add"}} >
-    <div class="vl-modal-dialog__content">
-      {{!-- {{#if this.savePieces.isRunning}} --}}
-      {{#if false}}
-        <WebComponents::VlLoader @text={{t "documents-loading-text"}} />
-      {{else}}
-        <WebComponents::AuAlert @skin="warning" @message="Gelieve nog geen documenten te uploaden, dit werkt nog niet"></WebComponents::AuAlert>
-        <FileUploader
-          @multipleFiles={{true}}
-          {{!-- @uploadedFileAction={{this.uploadPiece}} --}}
-        />
-        {{!-- <div class="upload-container">
+  <WebComponents::AuModal @resized={{this.isUploadModalResized}} @size="full-screen">
```

Comment:
Als ik escape duw op de modal sluit deze niet.
Als ik me niet vergis zat dit er vroeger wel in?

Reply:
in de vl-modal gaat dat insteken, maar in de au-modal zit dat nog niet in.

Reply:
Dit bekijken we best apart.
Ik heb een <ClickOutside> geprobeerd maar dan zit de css verkeerd.
en dan moet er ook een een verify ofzo bij in zodat je niet al je geuploade documenten kwijt bent als je per ongeluk ernaast klikt ...